### DISCUSSION TITLE
Why is there no lag in playback using opencv, but lag in web playback?

### DISCUSSION THREAD

**Discussion by zhuya1996** [mpromonet/webrtc-streamer#660] [Q&A]
![image](https://github.com/mpromonet/webrtc-streamer/assets/25482513/eaafcfb1-49da-4a0b-a187-5626f598b953)
![image](https://github.com/mpromonet/webrtc-streamer/assets/25482513/2d53443b-4ee1-442a-b02c-3875268699e9)
I changed it to true, opencv does not freeze, but the web side freezes, why is this?

**Comment by zhuya1996** [comment]
[My English is not good, what I mean is that when the web player reaches the i frame, it will pause and the frame rate will decrease.](https://github.com/mpromonet/webrtc-streamer/discussions/660#discussioncomment-8176738)

**Comment by mpromonet** [comment]
Hi,

As you see, this parameter is trying to keep duration between frame https://github.com/mpromonet/webrtc-streamer/blob/master/inc/VideoDecoder.h#L158.
This is important reading file, to play at normal speed, for video streaming it seems not useful, this why it is disable.

Best Regards,
Michel.

**Comment by zhuya1996** [comment]
In fact, it has played a role in RTSP, reducing the phenomenon of lagging, but it is not yet the most perfect
